### PR TITLE
add lookup fields section to docs

### PR DESCRIPTION
**Pull request by jumcorredorro**
## Description
Add lookup fields section to docs. Explaining new functionality of drip fields.

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue, bump the third digit of the version)
- [ ] New feature (non-breaking change that adds functionality, bump the second digit of the version)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality, bump the first digit of the version)
- [x] Documentation update (Don't bump the version of the project)

### CODE REVIEW

**Review by JorgeMichelena — APPROVED** [review]

File: docs/source/content/create.drip.rst [hunk 1]
```diff
@@ -16,11 +16,20 @@ On the other hand, when you click in the ``FIELD NAME OF USER`` input, you will
 
 Here you can relate the Drip to the corresponding ``Campaign``. Grouping several drips under a campaign.
 
+Lookup fields
+-------------
+The fields you can make a query are User model fields and fields of models related to the user model.
```
Comment (brunomichetti) [inline docs/source/content/create.drip.rst:21]:
```suggestion
You can make queries using the User's model fields and the fields of the models related to the user.
```

Comment (brunomichetti) [reply docs/source/content/create.drip.rst:21]: Added this suggestion because "The fields you can make a query" is missing something

**Review by brunomichetti — APPROVED** [review]
Approved with minor suggestion

File: docs/source/content/create.drip.rst (hunk 1 above)
Comment (jumcorredorro) [reply docs/source/content/create.drip.rst:21]: Sounds and looks better, thanks!

**Pull request merged**